pyuntl/untl_structure.py

```diff
@@ -429,11 +431,11 @@ def create_xml_file(self, untl_filename):
         """
         try:
             f = open(untl_filename, 'w')
-            f.write(self.create_xml_string().encode('utf-8'))
+            f.write(self.create_xml_string().decode('utf-8'))
```

Comment:
Is this right? I haven't gotten around to upgrading Python code yet, but it seems odd to me that an `encode` would turn to a `decode` with no other changes. What datatype is `create_xml_string()` returning?

Reply:
Well...`create_xml_string()` returns bytes, since UNTL metadata root object is byte type. So, I used decode().

Reply:
I'm keeping this one alive because of @ldko's comment which has not been addressed.